tests/brevitas/optim/test_sign_sgd.py

```diff
@@ -0,0 +1,310 @@
+"""
```

Comment:
Still confused about this license (not sure where my previous comment went)

Reply:
Similar to: https://github.com/Xilinx/brevitas/blob/master/src/brevitas/nn/quant_mha.py.